Hopin API Collection.postman_collection.json

```diff
@@ -0,0 +1,1153 @@
+                "com"
+              ],
+              "path": [
+                "v1",
+                "reports",
+                "{{reportId}}"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "Create Report for an event",
+          "request": {
+            "auth": {
+              "type": "bearer",
+              "bearer": [
+                {
+                  "key": "token",
+                  "value": "{{bearer_token}}",
+                  "type": "string"
+                }
+              ]
+            },
+            "method": "POST",
+            "header": [],
+            "url": {
+              "raw": "https://api.hopin.com/v1/events/{{eventId}}/reports/{{reportType}}",
+              "protocol": "https",
+              "host": [
+                "api",
+                "hopin",
+                "com"
+              ],
+              "path": [
+                "v1",
+                "events",
+                "{{eventId}}",
+                "reports",
+                "{{reportType}}"
+              ]
+            }
+          },
+          "response": []
+        }
+      ]
+    },
+    {
+      "name": "Tags",
+      "item": [
+        {
+          "name": "Returns the list of tags for a Booth",
+          "request": {
+            "auth": {
+              "type": "bearer",
+              "bearer": [
+                {
+                  "key": "token",
+                  "value": "{{bearer_token}}",
+                  "type": "string"
+                }
+              ]
+            },
+            "method": "GET",
+            "header": [],
+            "url": {
+              "raw": "{{ProdURL}}/v1/booths/{{boothId}}/tags",
+              "host": [
+                "{{ProdURL}}"
+              ],
+              "path": [
+                "v1",
+                "booths",
+                "{{boothId}}",
+                "tags"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "Returns the list of tags for a ScheduleItem",
+          "request": {
+            "auth": {
+              "type": "bearer",
+              "bearer": [
+                {
+                  "key": "token",
+                  "value": "{{bearer_token}}",
+                  "type": "string"
+                }
+              ]
+            },
+            "method": "GET",
+            "header": [],
+            "url": {
+              "raw": "{{ProdURL}}/v1/booths/{{scheduleId}}/tags",
+              "host": [
+                "{{ProdURL}}"
+              ],
+              "path": [
+                "v1",
+                "booths",
+                "{{scheduleId}}",
+                "tags"
+              ]
+            }
+          },
+          "response": []
+        }
+      ]
+    },
+    {
+      "name": "Data Subscription",
+      "item": [
+        {
+          "name": "Create a new data subscription",
```

Comment:
`for user's organization's event`? or maybe `for an event`? we're not really consistent here